client/components/PubReleaseDialog/PubReleaseDialog.js

```diff
@@ -68,7 +68,7 @@ const PubReleaseDialog = (props) => {
 			draftKey: historyData.latestKey,
 			makeDraftDiscussionsPublic: makeDraftDiscussionsPublic,
 		})
-			.then(({ release }) => {
+			.then((release) => {
```

Comment:
There is a corresponding change in the Releases `api.js` file to return the bare Release object.